data/sles4sap/qe_sap_deployment/mr_test_gcp.yaml

```diff
@@ -0,0 +1,32 @@
+provider: 'gce'
+apiver: 3
+terraform:
+  variables:
+    project: "ei-sle-qa-sap-8469"
+    region: "%PUBLIC_CLOUD_REGION%"
+    admin_user: "cloudadmin"
+    os_image: "%SLE_IMAGE%"
```

Comment:
I think it will be better to switch %SLE_IMAGE% for %QESAP_CLUSTER_OS_VER%

%SLE_IMAGE%  is filled in by provider.pm/terraform_aply whch I am planning to replace soon. 
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/91cf3603b32985ce2152674bcbf092bd2433b2e0/lib/publiccloud/provider.pm#L356

I see in your test PUBLIC_CLOUD_OS_IMAGE a well. Is this directly defined somewhere in Job Group ?

Sorry... this is the variable mess we were talking about last week :/

Reply:
Good question, PUBLIC_CLOUD_OS_IMAGE is set by default in job_template "&pc_defaults" (for example: https://openqaworker15.qa.suse.cz/admin/job_templates/21?), I did not use this variable and I am not sure if somebody still use it. It is useless in my test cases but still will be shown up in test "Setting".

Reply:
Question, I did not find the code for "%SLE_IMAGE% is filled in by provider.pm/terraform_aply whch I am replacing at the moment"  So I will keep the `os_image: "%SLE_IMAGE%"`

Reply:
PUBLIC_CLOUD_OS_IMAGE  - I think this is currently not used by anything... but let's leave it then. It was just a cleanup possibility. Once the whole deployment thing is done we can get together with Jozef and do the variable cleanup. 
 %SLE_IMAGE% - This is quite an entangled setup inherited from previous deployment code:

 This one overwrites terraform.tfvars with SLE_IMAGE:
 https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/33179656a8525b67d6cc75850b831cc94186622f/lib/publiccloud/provider.pm#L391

 and this variable is generated by "get_image_id()":
 https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/33179656a8525b67d6cc75850b831cc94186622f/lib/publiccloud/provider.pm#L365
 
 Currently both ways should work, ... but ... I can possibly reuse this variable so you don't need to change anything later. 
 Ok, please ignore this comment. I will try to make it work without breaking your stuff :)

Reply:
Sure, keeping `os_image: "%SLE_IMAGE%"`